logstash-core/src/main/java/org/logstash/instrument/witness/PluginsWitness.java

```diff
@@ -20,10 +30,12 @@ public PluginsWitness() {
         this.inputs = new ConcurrentHashMap<>();
         this.outputs = new ConcurrentHashMap<>();
         this.filters = new ConcurrentHashMap<>();
-     }
+        this.codecs = new ConcurrentHashMap<>();
```

Comment:
@original-brownbear - What are thoughts around optimizing non-hot code ?  I tend to leave the default value as-is since changing provides little value and it may send the wrong message to future developers.